docsteady/vcd.py

```diff
@@ -425,7 +428,8 @@ def get_ves(comp):
                     tsum = vby[1].split(':')
                     vbytmp.append(tsum[0])
                     verifying_ves.append(str(vby[0]))
-                tmpve['verifiedby'] = vbytmp
+                tmpve['verified_by'] = vbytmp  # was without _
```

Comment:
If the comment is needed can it say what caused the transition from verifiedby to verified_by?

Reply:
I wish I knew - I have no idea how this change happened - perhaps someine renamed it in Jira ....  it worked last year